src/test/resources/responses/rmapi/packages/get-created-package-response.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
this file and post-package-request.json probably shouldn't be in  rmapi/packages/ subfolder, because they are request/response for kb-ebsco-java module

Reply:
moved to other package